releng/platform-definitions/third-party.target

```diff
@@ -0,0 +1,125 @@
+		    <dependencies>
+			    <dependency>
+				    <groupId>com.sun.activation</groupId>
+				    <artifactId>jakarta.activation</artifactId>
+				    <version>2.0.1</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>com.sun.mail</groupId>
+				    <artifactId>jakarta.mail</artifactId>
+				    <version>2.0.1</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.adoptopenjdk</groupId>
+				    <artifactId>jemmy-awt-input</artifactId>
+				    <version>2.0.0</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.adoptopenjdk</groupId>
+				    <artifactId>jemmy-browser</artifactId>
+				    <version>2.0.0</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.adoptopenjdk</groupId>
+				    <artifactId>jemmy-core</artifactId>
+				    <version>2.0.0</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.adoptopenjdk</groupId>
+				    <artifactId>jemmy-swt</artifactId>
+				    <version>2.0.0</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.apache.aries.spifly</groupId>
+				    <artifactId>org.apache.aries.spifly.dynamic.bundle</artifactId>
+				    <version>1.3.4</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.eclipse.jetty.websocket</groupId>
+				    <artifactId>websocket-javax-server</artifactId>
+				    <version>10.0.7</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.eclipse.jetty.websocket</groupId>
+				    <artifactId>websocket-jetty-api</artifactId>
+				    <version>10.0.7</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.eclipse.jetty.websocket</groupId>
+				    <artifactId>websocket-jetty-server</artifactId>
+				    <version>10.0.7</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.hdrhistogram</groupId>
+				    <artifactId>HdrHistogram</artifactId>
+				    <version>2.1.12</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.lz4</groupId>
+				    <artifactId>lz4-java</artifactId>
+				    <version>1.8.0</version>
+				    <type>jar</type>
+			    </dependency>
+			    <dependency>
+				    <groupId>org.owasp.encoder</groupId>
+				    <artifactId>encoder</artifactId>
+				    <version>1.2.3</version>
+				    <type>jar</type>
+			    </dependency>
+		    </dependencies>
+		    <instructions><![CDATA[
+Bundle-Name:           Bundle derived from maven artifact ${mvnGroupId}:${mvnArtifactId}:${mvnVersion}
+Import-Package:        *;resolution:=optional
+Export-Package:        *
```

Comment:
These are different than the standard Tycho defaults, but match what was in use by the old maven plugin:  https://github.com/reficio/p2-maven-plugin#default-options